core/runtime/op_driver/joinset_driver.rs

```diff
@@ -157,4 +154,8 @@ impl<C: OpMappingContext> OpDriver<C> for JoinSetDriver<C> {
   fn len(&self) -> usize {
     self.pending_ops.borrow().len()
   }
+
+  fn stats(&self) -> OpInflightStats {
+    OpInflightStats::default()
+  }
```

Comment:
This means that stats are never collected for joinset driver?

Reply:
Correct -- that driver will hopefully go away soon once we're confident (maybe after 1.40?)